configs/env_config.py

```diff
@@ -5,4 +5,17 @@
 
 class EnvConfig(BaseConfig):
     def _get_config_value(self, variable_name):
-        return os.getenv(variable_name)
+        if variable_name in (
+            "AWS_REGION",
```

Comment:
Isnt AWS_REGION needed for the s3 download when running locally?

Reply:
AWS_REGION has been set in .env file locally, removed logic which is no longer required